tests/phpunit/tests/Discovery.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+namespace Google\Web_Stories\Tests;
+
+/**
+ * @coversDefaultClass \Google\Web_Stories\Discovery
+ */
+class Discovery extends \WP_UnitTestCase {
+	use Private_Access;
+
+	protected static $user_id;
+	protected static $story_id;
+	protected static $poster_attachment_id;
+
+	public static function wpSetUpBeforeClass( $factory ) {
+		self::$user_id = $factory->user->create(
+			[
+				'role' => 'administrator',
+			]
+		);
+
+		self::$story_id             = $factory->post->create(
+			[
+				'post_type'    => \Google\Web_Stories\Story_Post_Type::POST_TYPE_SLUG,
+				'post_title'   => 'Example title',
+				'post_status'  => 'publish',
+				'post_content' => 'Example content',
+				'post_author'  => self::$user_id,
+			]
+		);
+		self::$poster_attachment_id = $factory->attachment->create_object(
+			DIR_TESTDATA . '/images/test-image.jpg',
+			self::$story_id,
+			[
+				'post_mime_type' => 'image/jpeg',
+				'post_title'     => 'Test Image',
+			]
+		);
+		wp_maybe_generate_attachment_metadata( get_post( self::$poster_attachment_id ) );
+		set_post_thumbnail( self::$story_id, self::$poster_attachment_id );
+	}
+
+	public static function wpTearDownAfterClass() {
+		self::delete_user( self::$user_id );
+	}
+
+	public function setUp() {
+		parent::setUp();
+		$this->go_to( get_permalink( self::$story_id ) );
+	}
+
+	/**
+	 * @covers \Google\Web_Stories\Discovery::init
```

Comment:
Now thanks to `@coversDefaultClass` you can write it like this everywhere:

```suggestion
	 * @covers ::init
```

Reply:
Fixed.